### PR TITLE
`out` should be a directory

### PR DESCRIPTION
**Pull request by pkendall64**
After reading through the codebase and looking at the cmake files, it looks like the the final artefact `out` should actually be a directory that contains the `HDZGOGGLE` binary and the `HDZERO_GOGGLE-x.x.x.bin` firmware file that should be copied to the root directory if the SD card on the goggles for installation/upgrade.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -186,6 +186,7 @@ target_link_directories(${PROJECT_NAME} PRIVATE
 
 add_custom_command(TARGET ${PROJECT_NAME} POST_BUILD 
 											COMMAND size -A ${PROJECT_NAME}
+											COMMAND mkdir -p ${PROJECT_SOURCE_DIR}/out
 											COMMAND cp ${PROJECT_NAME} ${PROJECT_SOURCE_DIR}/out
 											COMMAND cp ${PROJECT_NAME} ${OS_APP_PATH}
 											COMMAND cd ${OS_APP_PATH} && cd ../../ && rm -f ./ota_app/hdzgoggle_app_ota*.tar && ./mkapp_ota && ./make_fw.sh
```
Comment (cruwaller) [inline CMakeLists.txt:192]: Could it be a good idea to change `cd ${OS_APP_PATH} && cd ../../` to `cd ${PROJECT_SOURCE_DIR}/mkapp/`

Comment (cruwaller) [reply CMakeLists.txt:192]: ...and add `/` in end of the move command below? `${PROJECT_SOURCE_DIR}/out` -> `${PROJECT_SOURCE_DIR}/out/`

Comment (pkendall64) [reply CMakeLists.txt]: Changes made as suggested 👍

**Review by cruwaller — APPROVED** [review]
Thanks PK!

**Pull request merged**